QamarDeen/QamarDeen/Model/Generated/Day+CoreDataProperties.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Day+CoreDataProperties.swift
+//  QamarDeen
+//
+//  Created by Mazyad Alabduljaleel on 12/25/16.
+//  Copyright © 2016 Batoul Apps. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+
+extension Day {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<Day> {
+        return NSFetchRequest<Day>(entityName: "Day");
+    }
+
+    @NSManaged public var hijriDate: NSDate?
```

Comment:
I was planning on following the legacy app footsteps, by storing the start of the day in UTC. It will require preprocessing the `NSDate` to prune any time information.

Personally, I would think storing a literal string would be the most predictable way, then we simply add some parsing on top.